Sources/Datadog/Tracing/Utils/ActiveSpanUtils.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import os.activity
+
+/// Helper class to get the current Span
+internal struct ActiveSpanUtils {
+    static let RTLD_DEFAULT = UnsafeMutableRawPointer(bitPattern: -2)
+    static let sym = dlsym(RTLD_DEFAULT, "_os_activity_current")
+    static let OS_ACTIVITY_CURRENT = unsafeBitCast(sym, to: os_activity_t.self)
```

Comment:
actually we can do these private API stuff in Obj-C: you can check `Sources/_Datadog_Private` folder

Reply:
Yes, its true, probably using directly Obj-c here and using our own wrapper  is much more clear than all this hacky code. I will work on it

Reply:
I moved this code to _Datadog_Provate modle